http/org.wso2.carbon.transport.http.netty/src/main/java/org/wso2/carbon/transport/http/netty/sender/ClientRequestWorker.java

```diff
@@ -16,8 +16,6 @@
 
```

Comment:
Please note that this class is there only as reference. Will remove it in the next refactor.